setup.py

```diff
@@ -52,16 +52,16 @@ def get_version(*file_paths):
     classifiers=[
         'Development Status :: 3 - Alpha',
         'Framework :: Django',
-        'Framework :: Django :: 1.8',
-        'Framework :: Django :: 1.9',
-        'Framework :: Django :: 1.10',
+        'Framework :: Django :: 2.0',
```

Comment:
Please add Django 1.11 here also (at least, I hope it's supported and working...)

Reply:
Yes.